docs/zopebook/Sessions.rst

```diff
@@ -1,6 +1,27 @@
 Session Management
 ##################
 
+Sessions in Zope have historically been server side. They are simple to use, 
+and (usually) only rely on one cookie that contains the browser id. However, 
+server side sessions are hard to scale in horizontal deployments where you are 
+using ZEO, or have many Zope servers with read only ZODBs. Also the historical 
+backend implementation of sessions 
+`tempstorage <https://github.com/zopefoundation/tempstorage>`_ is broken and 
+deprecated so server side sessions cannot be used 'as is' anymore in Zope 4+.
+
+For this reason, and especially if you need only very few small session values, 
+consider using cookies as a replacement. The API for that would be 
+``REQUEST.get()`` and ``RESPONSE.setCookie()``. Plone has 
+`a bit more documentation on how to work with cookies <https://docs.plone.org/develop/plone/sessions/cookies.html>`_.
+
+If you still need server side sessions (perhaps to migrate a old project) there 
```

Comment:
Rewrite that sentence please. The current wording sounds as if server side sessions are discouraged or not used in newer deployments:

```
If you need server side sessions see the section...
```

Reply:
[x] done (push to come)